### DISCUSSION TITLE
why rust?

### DISCUSSION THREAD

**Discussion by sunshaobin** [Brooooooklyn/canvas#339] [Q&A]
I  understand this package based on skia and export api for node environment, but why not node c++ extend.

**Comment by Brooooooklyn** [comment]
`Skia` can not do the whole Canvas jobs, we still need `cssparser` ,`regex` `base64` etc, which provided by `crates`. But C++ lack of package management.

**Reply by sunshaobin** [reply]
I see，thanks.
